video/out/mac/common.swift

```diff
@@ -395,7 +406,58 @@ class Common: NSObject {
     }
 
     func updateICCProfile() {
-        log.sendWarning("updateICCProfile not implemented")
+        guard let windowColorSpace = window?.screen?.colorSpace else {
+            log.sendWarning("Couldn't update ICC Profile, no color space available")
+            return 
+        }
+
+        colorSpace = windowColorSpace
+
+        if #available(macOS 10.11, *) {
+
+            // Experimental #HDR on #macOS        
+            var name = "" as CFString
+
+// HDR is supported fully only on macOS 10.15 and higher
+// some of the HDR color spaces are available on 10.14 though
+#if HAVE_MACOS_10_15_FEATURES
+                switch (mpv?.macOpts.macos_hdr_csp)
```

Comment:
the indentation is wrong here.

Reply:
Fixed it! Also seems that even though some color spaces are declared as supported by 10.14 SDK they are available only on 10.15+. Travis didn't let me. So put most of those inside `HAVE_MACOS_10_15_FEATURES`.